README.md

```diff
@@ -361,20 +319,9 @@ Now that you have the required infrastructure, it's safe to deploy the Kogito Da
 
 ### Install Data Index Service
 
-#### Retrieve Kafka and Infinispan internal urls
-
-Having [installed](#installation) the Kogito Operator, create a new `Kogito Data Index` resource using the services URIs from Infinispan and Kafka:
-
-```bash
-$ oc get svc -l app=infinispan-server
-
-NAME                TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)                                                                      AGE
-infinispan-server   ClusterIP   172.30.193.214   <none>        7600/TCP,8080/TCP,8181/TCP,8888/TCP,9990/TCP,11211/TCP,11222/TCP,57600/TCP   4m19s
-```
-
-In this example, the Infinispan Server service is `infinispan-server:11222`.
+#### Retrieve Kafka internal url
 
-Then grab the Kafka cluster URI:
+Having [installed](#installation) the Kogito Operator, create a new `Kogito Data Index` resource using the Kafka service URI:
```

Comment:
Shouldn't that be in "Install Data Index Service with Kogito CLI" part ?

And here write: "To deploy the data index service, we will need first to retrieve the Kafka Service URI:"

Reply:
Thanks, I rephrased!